app/src/main/java/ch/epfl/qedit/model/answer/MatrixFormat.java

```diff
@@ -13,10 +13,15 @@
     private int tableColumnsNumber = 1;
     private int maxCharacters = 5;
     private String hintString;
-    private String id = MatrixFragment.MATRIXID;
+    private String id = MatrixFragment.MATRIX_ID;
```

Comment:
I think that the ID is not really necessary, we removed it in our branch. Do you rely on it somewhere?

Reply:
No, I just renamed it because it was detected as a typo and it triggered me. You can remove it!